opm/material/fluidsystems/blackoilpvt/Co2GasPvt.hpp

```diff
@@ -402,4 +458,43 @@ class Co2GasPvt
 
 } // namespace Opm
 
+namespace Opm::gpuistl{
+    template<class Scalar, class Params, class GPUContainer>
+    Co2GasPvt<Scalar, Params, GPUContainer>
+    move_to_gpu(Co2GasPvt<Scalar> cpuCo2){
+        return Co2GasPvt<Scalar, Params, GPUContainer>(move_to_gpu<Scalar, std::vector<Scalar>, GPUContainer>(cpuCo2.getParams()),
```

Comment:
Hard to read, suggest putting move_to_gpu (or moveToGpu as I prefer...) on a separate line, and align the following arguments with that, so that every argument to the Co2GasPvt constructor call has the same indent.

Same for the makeView().